### DISCUSSION TITLE
I want the recent commits of all repositories that I am following

### DISCUSSION THREAD

**Discussion by HiGarfield** [community/community#66208] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Viewing the latest commits in all the repositories I follow on GitHub is an important way to stay updated on development progress and bug fixes. Losing this functionality means I have to manually check dozens of repositories every day to see if there are any new commits, which is extremely inconvenient for me.

**Comment by blank3times** [comment]
I like this suggestion better than the other ones since you can remove recommendations using filters (so the "bring back the old one" isn't completely based in reason), but we certainly can't _add_ functionality. So this is also why it's definitely not a duplicate of any "make the two separate" requests.
